PostPairsTrading.md

```diff
@@ -282,21 +259,20 @@ In this instance, we can see that the spread (purple line) is positive and above
 
 > 💡 Signal windows play a pivotal role in implementing Pairs Trading strategies. They serve as indicators for determining when to execute buy and sell actions, acting as arbitrary thresholds that guide our algorithm's decision-making process. These windows are derived from the variance of our data, representing a static variance assumption due to our consideration of a time-independent cointegrated series.
 
-## Conclusion and Future Work
+## Conclusion
 
-In this post, we have provided a comprehensive overview of Pairs Trading, covering its implementation and intricacies in KDB+/Q.
+In this post, we have provided a comprehensive overview of the implementation of the Pairs Trading strategy in KDB+/Q, contextualized within the Tick Architecture. Here are some key takeaways:
+* The Tick Architecture allows us to handle both historical and real-time data.
+* By leveraging historical data, we were able to easily identify cointegrated pairs, reusing libraries from the Python ecosystem via PyKX when needed.
+* Q is very expressive and the implementation of the Linear Regression logic for producing the spread model is straightforward.
+* Integrating a real-time component and connecting it with a dashboard is simple and efficient.
 
-We have discussed:
+More generally, and although we couldn't delve into all the details in this post, we'd like to emphasize the three major selling points of KDB+/Q. First, it can process large amounts of data in a very short time with a small memory footprint, allowing us to monitor thousands of pairs simultaneously. Secondly, Q code is highly concise, enabling us to implement all the components in the diagram in less than 100 lines of code. Finally, the technology is highly flexible, allowing us to easily adapt to other scenarios beyond Pairs Trading.

```

Comment:
```suggestion
More generally, and although we couldn't get into all the details in this post, we'd like to emphasize the three major selling points of KDB+/Q. First, it can process large amounts of data in a very short time with a small memory footprint, allowing us to monitor thousands of pairs simultaneously. Secondly, Q code is highly concise, enabling us to implement all the components in the diagram in less than 100 lines of code. Finally, the technology is highly flexible, allowing us to easily adapt to other scenarios beyond Pairs Trading.

```

Reply:
Ok, you don't like _delve_ at all 😆